### PR TITLE
Added RuleId as another optional instance variable

### PR DESCRIPTION
**Pull request by anu20890**
Added RuleId as another optional variable in case there are ids assigned to different rules stored in databases or files.

### CODE REVIEW

**Comment by abbasc52** [comment]
@anu20890 can you provide more details on use-case for RuleId?

**Comment by anu20890** [comment]
@abbasc52 
The use case is, Rule defined in database can have both name as well as Id, and after successful evaluation, We have to retrieve both. So if `Rule` model class could have another instance variable `RuleId `to hold that id, it will be helpful.

File: src/RulesEngine/Models/Rule.cs [hunk 1]
```diff
@@ -17,6 +17,14 @@ namespace RulesEngine.Models
...
     {
         public string RuleName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the custom property or tags of the rule.
+        /// </summary>
+        /// <value>
+        /// The property of the rule.
+        /// </value>
+        public Dictionary<string, object> RuleProperty { get; set; }
```
Comment (abbasc52) [inline src/RulesEngine/Models/Rule.cs:27]: How about calling it just `Properties`?

Comment (anu20890) [reply src/RulesEngine/Models/Rule.cs]: Renamed the variable to `Properties`

**Review by abbasc52 — APPROVED** [review]

**Pull request merged**